canonical-web-and-design/jaas-dashboard
# Correct weird spacing in models details tables

## Done

Correct weird spacing in models details tables

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Verify spacing in models details is consistent across FF and Chrome

## Details

Fixes #207, Fixes #208
